Sources/swift-composable-architecture-benchmark/Dependencies.swift

```diff
@@ -25,7 +25,7 @@ let dependenciesSuite = BenchmarkSuite(name: "Dependencies") { suite in
 
 private struct BenchmarkReducer: ReducerProtocol {
   @Dependency(\.someValue) var someValue
-  func reduce(into state: inout Int, action: Void) -> Effect<Void, Never> {
+  func reduce(into state: inout Int, action: Void) -> EffectPublisher<Void, Never> {
```

Comment:
```suggestion
  func reduce(into state: inout Int, action: Void) -> EffectTask<Void> {
```
This is the product of a reducer, it should be `EffectTask` for now.